package.json

```diff
@@ -28,6 +28,7 @@
     "test": "npm run lint:quiet && npm run test:compile && npm run mocha",
     "test:client-integration": "bash ./scripts/client-integration-tests/run.sh",
     "test:compile": "tsc --project tsconfig.compile.json",
+    "test:management": "npm run lint:quiet && npm run test:compile && nyc mocha 'src/management/*.spec.{ts,js}'",
```

Comment:
This is probably unecessary since these unit tests are covered by `npm run mocha`. All these other e2e tests have special commands because they use code from `./scripts` and do things that are not appropriate for unit tests (ie calls to prod with credentials)